bitnami/keycloak/values.yaml

```diff
@@ -103,7 +103,7 @@ diagnosticMode:
 image:
   registry: docker.io
   repository: bitnami/keycloak
-  tag: 24.0.5-debian-12-r3
+  tag: latest
   digest: ""
   ## Specify a imagePullPolicy
```

Comment:
Perhaps it would be interesting to relate the versions between the application and the helm chart